src/bin/flamegraph.rs

```diff
@@ -31,15 +32,61 @@ struct Opt {
         )
     )]
     colors: Palette,
+
     /// set background colors. Gradient choices are yellow (default), blue, green, grey; flat colors use "#rrggbb"
     #[structopt(long = "bgcolors")]
     bgcolors: Option<BackgroundColor>,
+
     /// colors are keyed by function name hash
     #[structopt(long = "hash")]
     hash: bool,
+
     /// use consistent palette (palette.map)
     #[structopt(long = "cp")]
     cp: bool,
+
+    /// Change title text
+    #[structopt(name = "title", long = "title")]
+    title: String,
```

Comment:
Shouldn't this also be an `Option<String>`, given that the user doesn't _have_ to give one?

Reply:
No, according to original implementation there is default, but I missed it.